datalad/support/gitrepo.py

```diff
@@ -1999,6 +1999,36 @@ def tag(self, tag):
             '', ['git', 'tag', str(tag)]
         )
 
+    def get_tags(self, names_only=False):
```

Comment:
Yeah, yeah....communists....everything has to look the same.... I like it. Pushed.